scripts/npm-safe-publish.sh

```diff
@@ -4,41 +4,47 @@
 # If it doesn't exist, check that we're on the main branch, the working directory is clean, and the current commit
 # is tagged with the requested version
 
+set -euo pipefail
```

Comment:
@JamieSinn / @ajwootto can you review the changes to this script to support deploying to deprecated packages?

Reply:
yep